<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinAutoDialog</class>
 <widget class="QDialog" name="SkinAutoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skin auto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainFrame.qrc">
    <normaloff>:/MainFrame/Resources/ModelEditor.png</normaloff>:/MainFrame/Resources/ModelEditor.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Squelette</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="loadSkel">
         <property name="text">
          <string>Charger squelette</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="skelName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="bonesList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okSkel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainFrame.qrc"/>
 </resources>
 <connections/>
</ui>
